driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -140,6 +145,30 @@ OperationResult executeFind(final BsonDocument operation) {
                 new BsonArray(iterable.into(new ArrayList<>())));
     }
 
+    OperationResult executeDistinct(final BsonDocument operation) {
```

Comment:
Added support for a bunch more operations that are used in the versioned API tests.